scripts/speech_llm/oomptimizer.py

```diff
@@ -0,0 +1,578 @@
+#!/usr/bin/env python
```

Comment:
r.e. the problem of real memory consumption was larger than the result from oomptimizer ,  do we have any update ? if that was the case, how can we use oomptimizer?

Reply:
Haven't looked into it further yet. At the moment I have to multiply the batch sizes obtained from oomptimizer by ~0.6 to  get it working.